homeassistant/components/utility_meter/services.yaml

```diff
@@ -1,35 +1,69 @@
 # Describes the format for available switch services
 
 reset:
+  name: Reset
   description: Resets the counter of an utility meter.
   fields:
     entity_id:
```

Comment:
Entity_id should be replaced by a service target.

```yaml
reset:
   name: Reset
   description: Bla
   target:
```


(applies to other occurrences as well).

Reply:
Thanks for the heads-up

Reply:
fixed

Yet I must have something wrong in the component itself because the filter works with other integrations, just not with utility_meter...

Reply:
so it seems I would have needed to have a config_flow in order to have en entity_registry entry. Since the integration is internal that is not possible, therefore this PR is good to go with just sensor filter